finder.py

```diff
@@ -322,10 +322,26 @@ def find_match(slug):
 		timestamp = datetime.datetime.strptime(clip_info["vod_data"]["created_at"], '%Y-%m-%dT%H:%M:%SZ')
 		timestamp = int(timestamp.replace(tzinfo=datetime.timezone.utc).timestamp())
 		timestamp += clip_info["vod"]["offset"]
-		new_diff = (timestamp - match["start_time"]) // 60
+		new_diff = (timestamp - best_match["start_time"]) // 60
 		if new_diff > 0:
 			result["better_minutes_diff"] = new_diff
 
+	# Get league clip
+	match_details_url = f"https://api.opendota.com/api/matches/{best_match['match_id']}"
+	try:
+		response = requests.get(match_details_url)
+		if response.status_code != 200:
+			raise OpendotaApiException(message=f"Match id<{best_match['match_id']} not found.")
+		match_details = response.json()
+		if 'tier' in match_details['league']:
+			if match_details['league']['tier'] == 'premium':
+				result['league_name'] = match_details['league']['name']
+				result['team_rad'] = match_details['radiant_team']['name']
+				result['team_dire'] = match_details['dire_team']['name']
+
+	except json.decoder.JSONDecodeError as e:
+		raise OpendotaApiException from e
```

Comment:
Similar to above, if this request fails, we shouldn't throw an exception, because this is optional data and if it is missing we can still show what we would've shown (pretend that we know that this isn't a league game). We should probably just do a `pass` on a JSONDecodeError

Reply:
Fixed in commit a603963c341d9b45c309631e35444847920aed8f